src/mongoose_cleaner.erl

```diff
@@ -75,15 +77,15 @@ cleanup_modules(Node) ->
     LockRequest = {LockKey, self()},
     C = fun () -> run_node_cleanup(Node) end,
     Nodes = [node() | nodes()],
-    Retries = 1,
+    Retries = 10,
     case global:trans(LockRequest, C, Nodes, Retries) of
         aborted ->
-            ?LOG_INFO(#{what => cleaner_trans_aborted,
-                        text => <<"mongoose_cleaner failed to get global lock">>,
-                        lock_key => LockKey}),
-            {ok, aborted};
+            ?LOG_ERROR(#{what => cleaner_trans_aborted,
```

Comment:
Maybe a warning would be better, as with many nodes IMO this is quite likely to happen, and it doesn't have any immediate bad consequences, like lost data, lack of functionality etc.